wpmn-loader.php

```diff
@@ -145,6 +145,9 @@ private function includes() {
 		if ( defined( 'WP_CLI' ) && WP_CLI ) {
 			require $this->plugin_dir . 'includes/classes/class-wp-ms-networks-cli.php';
 		}
+
+		// REST endpoint class
+		require $this->plugin_dir . 'includes/classes/class-wp-ms-networks-api.php';
```

Comment:
The file should be called `class-wp-ms-rest-networks-controller.php`, like the class name.